dexbot/strategies/relative_orders.py

```diff
@@ -52,19 +74,46 @@ def __init__(self, *args, **kwargs):
         self.manual_offset = self.worker.get('manual_offset', 0) / 100
         self.order_size = float(self.worker.get('amount', 1))
         self.spread = self.worker.get('spread') / 100
+        self.is_reset_on_partial_fill = self.worker.get('reset_on_partial_fill', True)
+        self.partial_fill_threshold = self.worker.get('partial_fill_threshold', 30) / 100
+        self.is_reset_on_market_trade = self.worker.get('reset_on_market_trade', False)
+        self.is_reset_on_price_change = self.worker.get('reset_on_price_change', False)
+        self.price_change_threshold = self.worker.get('price_change_threshold', 2) / 100
+        self.is_custom_expiration = self.worker.get('custom_expiration', False)
+
+        if self.is_custom_expiration:
+            self.expiration = self.worker.get('expiration_time', self.expiration)
+
+        self.last_check = datetime.now()
+        self.min_check_interval = 8
 
         self.buy_price = None
         self.sell_price = None
 
         self.initial_balance = self['initial_balance'] or 0
         self.worker_name = kwargs.get('name')
         self.view = kwargs.get('view')
-        self.check_orders()
+
+        # Check for conflicting settings
+        if self.is_reset_on_price_change and not self.is_center_price_dynamic:
+            self.log.error('reset_on_price_change requires Dynamic Center Price')
+            self.disabled = True
+        self.update_orders()
```

Comment:
Is there a reason you changed check_orders() to update_orders()?

Reply:
Yes, because relative orders strategy is supposed to remove orders on exit, we can assume on initializing that we don't have any orders anyway, so there is no sense in checking something that should not be exist.

Reply:
Well you can actually keep the orders on the market with relative orders on exit. If you simply quit the program when relative orders is running, it leaves the orders on the market.

I just feel it's a waste to cancel those orders on start up.

Reply:
Yes, I agree. Furthermore, we can add an option into GUI like "Keep orders on exit" to let the user decide should the bot keep orders or not.